lib/factory_girl/strategy_syntax_method_registrar.rb

```diff
@@ -25,6 +25,10 @@ def define_list_strategy_method
       strategy_name = @strategy_name
 
       define_syntax_method("#{strategy_name}_list") do |name, amount, *traits_and_overrides, &block|
+        unless amount.is_a?(Integer)
```

Comment:
What do you think about asking for `respond_to(:times)` here instead? It's probably a minor difference but we don't care what the type is really, only that it responds to the `times` method.